pyshelf/link_manager.py

```diff
@@ -68,3 +68,19 @@ def assign_single(self, artifact_path):
         ]
 
         self.context.link_list = link_list
+
+    def assign_metadata(self, metadata_path):
```

Comment:
Actually I kind of take this back.  I think what we really want is for a related link to come back for the artifact that it is related to.  We should be able to build this into assign_single also